agentops/llm_tracker.py

```diff
@@ -417,6 +418,24 @@ def generator():
 
         return response
 
+    def _handle_response_ollama(self, response, kwargs, init_timestamp):
+        self.llm_event = LLMEvent(init_timestamp=init_timestamp, params=kwargs)
+
+        self.llm_event.end_timestamp = get_ISO_time()
+
+        self.llm_event.model = response["model"]
+        self.llm_event.returns = response
+        self.llm_event.agent_id = check_call_stack_for_agent_id()
+        self.llm_event.prompt = kwargs["messages"]
+        self.completion = response["message"]
+
+        # ollama doesn't return the tokens
+        # still looking at how to find this
+        self.llm_event.prompt_tokens = 0
+        self.llm_event.completion_tokens = 0
+
+        return response
```

Comment:
You need to also call client.record to send the completed LLM Event to the API server:

Add `self.client.record(self.llm_event)` before you return the response

Reply:
Thanks for pointing this out. Adding this fixed the UI also.